server/db/controllers/usersController.js

```diff
@@ -0,0 +1,36 @@
+const User = require("../../models/usersModel");
+
+module.exports.create = function(req, res) {
+  const userObj = req.body;
+
+  User.addUser(userObj)
+    .then(() => {
+      res.sendStatus(201);
+    })
+    .catch(err => {
+      console.error(`An error occured while creating a user: ${err}`);
+      res.sendStatus(500);
+    });
+};
+
+module.exports.retrieve = function(req, res) {
+  User.getUser()
+    .then(data => {
+      userData = data[0]["dataValues"];
```

Comment:
Let's refactor this manipulation into the Model, as it seems like Model logic rather than controller logic